src/OrchardCore.Modules/OrchardCore.AdminDashboard/Services/DashboardPartContentTypeDefinitionHandler.cs

```diff
@@ -0,0 +1,84 @@
+using System.Text.Json.Nodes;
+using OrchardCore.AdminDashboard.Models;
+using OrchardCore.ContentManagement.Metadata.Records;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.ContentTypes.Events;
+using OrchardCore.Modules;
+
+namespace OrchardCore.AdminDashboard.Services;
+
+public sealed class DashboardPartContentTypeDefinitionHandler : ContentDefinitionHandlerBase
```

Comment:
Each method should have a comment describing what it does:

`// Ensure existing type-part named DashboardPart have the setting ...` for instance
Because it's hard sometimes to distinguish between what is for handling the previous way or for all cases.

Reply:
done